libs/packages/sam-formly/src/lib/formly-filters/advanced-filters/advanced-filters.component.ts

```diff
@@ -152,7 +165,61 @@ export class AdvancedFiltersComponent {
         ]
       }
     ];
-    const updateField: FormlyFieldConfig[] = [...filedGroup, ...modalFields];
+
+    const showInactiveGroup: FormlyFieldConfig[] = [
+      {
+        template: '<hr/>'
+      },
+      {
+        fieldGroup: [
+          {
+            key: 'showInactive',
+            type: 'checkbox',
+            defaultValue: this.showInactive,
+            templateOptions: {
+              label: 'Show Inactive Filter Values (Indicated by *)',
+              hideOptional: true,
+              id: 'moreFilterSelectAll',
+              textAlignEnd: true
+            },
+            hooks: {
+              onInit: field => {
+                let isOnload = true;
+                const form = field.parent.formControl;
+                form
+                  .get('showInactive')
+                  .valueChanges.pipe(
+                    startWith(form.get('showInactive').value),
+                    tap(showInactiveValue => {
+                      this.showInactive = showInactiveValue;
+                      isOnload = false;
+                    })
+                  )
+                  .subscribe();
+              }
+            }
+          }
+        ]
+      }
+    ];
+    let updateField: FormlyFieldConfig[];
+    if(this.isInactiveValueFieldShown){
+      updateField = [{
+        key: 'filterToggle',
+        fieldGroup: [...filedGroup, {
+          key: 'filters',
+          fieldGroup: [...modalFields]
+        }]
+      }, ...showInactiveGroup];
+    } else {
+      updateField = [{
+        key: 'filterToggle',
+        fieldGroup: [...filedGroup, {
+          key: 'filters',
```

Comment:
Could we make this filters string a readonly text for the component? It's used in multiple places throughout the component, and could cause weird bugs in case of future typos.

Reply:
Added variable `filtersFieldGroupKey`